react/shippingStateMachine/shippingStateMachine.ts

```diff
@@ -141,7 +141,7 @@ const shippingStateMachine = Machine<
             invoke: {
               src: 'tryToEditReceiverInfo',
               onDone: {
-                target: '#shipping.selectDeliveryOption',
+                target: '#shipping.done',
```

Comment:
this might be intended but the flow to edit the receiver name on the address would always redirect the user to the payment step?

in case you went back to the shipping just to edit the receiver name (the flow being: other step -> edit shipping -> sla selection list -> edit receiver name -> payment) would it be desirable to redirect the user back to the SLA selection list or to send the user to payment? cc @augustocb

Reply:
I discussed with @augustocb and the intended flow will always be to send the user to the payment step, because the address is already complete and there is no need for more actions from the user.